src/fidesops/ops/service/authentication/authentication_strategy_oauth2_base.py

```diff
@@ -216,7 +174,7 @@ def _validate_and_store_response(
         # This alternate way of specifying the expiration is handled
         # by the optional expires_in field of the OAuth2AuthenticationConfiguration
 
-        expires_in = response.get("expires_in") or self.expires_in
+        expires_in = self.expires_in or response.get("expires_in")
```

Comment:
Switching order to be able to override expiration even if one is provided in the token response.